agda-tac/src/main.rs

```diff
@@ -30,8 +34,21 @@ async fn main() {
     let file = match args.file {
         Some(file) => file,
         None => {
-            eprintln!("No input file specified.");
-            std::process::exit(1);
+            println!("No input file specified.");
+            let agda_tac_dir = dirs::home_dir()
+                .expect(FAIL_RETRIEVE_HOME)
+                .join(".agda-tac");
+            let file_path = agda_tac_dir
+                .join("Nameless.agda")
+                .into_os_string()
+                .into_string()
+                .unwrap();
```

Comment:
`expect(bad dir name)` or something. The point is to show error message instead of crashing